src/constants.js

```diff
@@ -330,6 +331,10 @@ export const SIDEBAR_VIEW_METADATA: 'metadata' = 'metadata';
 export const SIDEBAR_VIEW_ACTIVITY: 'activity' = 'activity';
 export const SIDEBAR_VIEW_VERSIONS: 'versions' = 'versions';
 
+/* ------------------ Grid View ---------------------- */
+export const GRID_VIEW_MAX_COLUMNS: 7 = 7;
```

Comment:
It seems odd that this needs to be shared. It seems like it should either be passed in from the parent component (controlled) or it should live in the slider component (uncontrolled).